src/gutter-container.coffee

```diff
@@ -0,0 +1,131 @@
+{Model} = require 'theorist'
+{CompositeDisposable, Emitter} = require 'event-kit'
+{Gutter} = require './gutter'
+
+# This class encapsulates the logic for adding and modifying the gutters on a
+# TextEditor.
+
+module.exports =
+class GutterContainer extends Model
```

Comment:
I don't think you should extend `Model`. I know we do it elsewhere but eventually I want to rip it all out and go simpler.